backend/src/db/migrations/20200213230248-add_unique_index_to_image_url.js

```diff
@@ -0,0 +1,53 @@
+import { getDriver } from '../../db/neo4j'
+
+export const description = `
+  We introduced a new node label 'Image' and we need a primary key for it. Best
+  would probably be the 'url' property which should be unique and would also
+  prevent us from overwriting existing images.
+`
+
+export async function up(next) {
+  const driver = getDriver()
+  const session = driver.session()
+  const transaction = session.beginTransaction()
+
+  try {
+    // Implement your migration here.
+    await transaction.run(`
+      CREATE CONSTRAINT ON ( image:Image ) ASSERT image.url IS UNIQUE
+    `)
+    await transaction.commit()
+    next()
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error)
+    await transaction.rollback()
+    // eslint-disable-next-line no-console
+    console.log('rolled back')
+  } finally {
+    session.close()
+  }
+}
+
+export async function down(next) {
+  const driver = getDriver()
+  const session = driver.session()
+  const transaction = session.beginTransaction()
+
+  try {
+    // Implement your migration here.
+    await transaction.run(`
+    DROP CONSTRAINT ON ( image:Image ) ASSERT image.url IS UNIQUE
+    `)
+    await transaction.commit()
+    next()
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error)
+    await transaction.rollback()
+    // eslint-disable-next-line no-console
+    console.log('rolled back')
+  } finally {
+    session.close()
+  }
+}
```

Comment:
why do we need a migration for this? can't we just run `prod:migrate init` after having added a `model` for it?

Reply:
I thought we would only run `db:migrate up` after a helm upgrade? This is a training for us to create migrations when we add new constraints.